### ISSUE TITLE
Problem share file

### ISSUE THREAD

**Issue by D1mar0n** [sourcelocation/DebToIPA#11]
Hello! Once the file is converted, xxxx.ipa cannot be saved elsewhere or shared through installed applications. Please correct.

iPhone 12 Pro / iOS 14.2 / Unc0ver 8.0.2

**Comment by sourcelocation** [comment]
Hi, I need more context on this issue (screenshots/videos etc)

**Comment by D1mar0n** [comment]
Hello!  Thanks for the answer.  Yesterday, in the "Jailbreak" mode, I converted the file, everything turned out without errors.  When I tried to share this file via email or save it to "Files" and "Filza", nothing worked... After exiting "Jailbreak" mode, I managed to share the file through the mail application, but I can't save the file.  Video review attached here.

 I also want to ask you to add the ability to open files with the .deb extension (by clicking on such a file and choosing "Open in") directly in the DebToIpa application.  Thank you in advance for your response.



Enviado desde iPhone
